cellrank/kernels/_monge_kernel.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any, Tuple, Union, Literal, Callable, Optional, Sequence
+
+from moscot.problems.time import TemporalNeuralProblem
```

Comment:
There will need to be a try except statement, since we don't want to require `moscot` to be installed.